pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -337,6 +340,14 @@ public void setErrorOnColumnBuildFailure(boolean errorOnColumnBuildFailure) {
     _errorOnColumnBuildFailure = errorOnColumnBuildFailure;
   }
 
+  public void setEnablePreProcess(boolean enablePreProcess) {
+    _enableSegmentPreprocess = enablePreProcess;
+  }
+
+  public boolean isEnablePreProcess() {
+    return _enableSegmentPreprocess;
+  }
+
```

Comment:
Don't add them, and make the field final. `IndexLoadingConfig` shouldn't be modified

Reply:
it will be set in the `init()` method so cannot use final unless we explicitly set this field